web/src/containers/IncentivePayments.js

```diff
@@ -36,18 +34,31 @@ class IncentivePayments extends Component {
       <Fragment>
         {years.map(year => (
           <Fragment key={year}>
-            <h3 id={thId(year)}>{t('ffy', { year })}</h3>
-            <table className="table-cms table-fixed">
+            <table className="budget-table">
+              <caption className="ds-u-visibility--screen-reader">
+                {t('ffy', { year })}
```

Comment:
Should we have an issue for moving this table header out of this cell and into an `<h#>` like the other tables?

Reply:
I'm a little confused, tbh. It seems like most of the mocks that @austinhernandez has going actually move the year _into_ the corner cell as a standardization. But then there are also tables with multiple years so... I don't even know any more.